content/workshops/rhel_8/exercise1.0.adoc

```diff
@@ -0,0 +1,236 @@
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
+
+=== Section 7: Accounts Module
+
+image::cockpit_06_accounts.png[caption="Figure 8: ", title='Web Console Accounts Module', 500]
+
+The accounts module allows you to create and modify local user accounts.
+
+=== Section 8: Services Module
+
+image::cockpit_07_services.png[caption="Figure 9: ", title='Web Console Services Module', 500]
+
+The services module allows basic administration of systemd objects, including system services.
+
+=== Section 9: Session Recording Module
+
+image::cockpit_08_sr.png[caption="Figure 10: ", title='Web Console Session Recording Module', 500]
+
+This module provides management of recorded terminal sessions. 
+
+=== Section 10: Applications Module
+
+image::cockpit_09_applications.png[caption="Figure 11: ", title='Web Console Applications Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
+
+=== Section 11: Diagnostic Reports Module
+
+image::cockpit_10_dr.png[caption="Figure 12: ", title='Web Console Diagnostic Reports Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
+
+=== Section 12: Kernel Dump Module
+
+image::cockpit_11_kd.png[caption="Figure 13: ", title='Web Console Kernel Dump Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
+
+=== Section 13: SELinux Module
+
+image::cockpit_12_selinux.png[caption="Figure 14: ", title='Web Console SELinux Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
+
+=== Section 14: Software Update Module
+
+image::cockpit_13_su.png[caption="Figure 15: ", title='Web Console Kernel Dump Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
+
+=== Section 15: Subscriptions Module
+
+image::cockpit_14_subscriptions.png[caption="Figure 16: ", title='Web Console Subscriptions Module', 500]
+
+The main screen that you see, when you first login to the web console is the system module.  It contains an overview of the system, including the below sections:
+
+=== Section 16: Terminal Module
+
+image::cockpit_15_terminal.png[caption="Figure 17: ", title='Web Console Terminal Module', 500]
+
+The terminal module provides command-line access to the system, similar to an SSH session.  Section 16.1 has some example commands to try if you'd like to test it out.
```

Comment:
Line 201 The system module is the main screen that displays upon logging in to the web console. It presents an overview of the system, including the following system characteristics: